### DISCUSSION TITLE
Limit Google Drive daily backup to WiFi

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3007] [Ideas]
Can we have an optional setting added to the Google Drive page to not upload the automatic daily backup using data and do it only when there is WiFi?

**Comment by jimk9** [comment]
This would be a good option for me. I'm running the backup manually as my data allowance is limited. Unfortunately I keep forgetting to do it. Old age...........

**Reply by Navid200** [reply]
Thanks.  I forgot I had created this.

**Reply by Navid200** [reply]
@jimk9 The option has been added to the latest Nightly:  https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.10.24
So, you can set xDrip to do daily backup to Google Drive only on WiFi.

**Reply by jimk9** [reply]
That's great, thanks
